.github/ISSUE_TEMPLATE/ios.md

```diff
@@ -0,0 +1,33 @@
+---
+name: iOS
+about: Mapbox Maps SDK for iOS
+
+---
+<!--
+Hello and thanks for contributing to the Mapbox Maps SDK for iOS! To help us diagnose your problem quickly, please:
+
+ - Include a minimal demonstration of the bug, including code, logs, and screenshots.
+ - Ensure you can reproduce the bug using the latest release.
+ - Only post to report a bug or request a feature; direct all other questions to: https://stackoverflow.com/questions/tagged/mapbox
+
+Start with a brief description below this line. -->
+
+### Steps to reproduce
+
+ 1.
+ 2.
+ 3.
+
+### Expected behavior
+
+
+
+### Actual behavior
+
+
+
+### Configuration
+
+**iOS versions:** 
+**Device/simulator models:** 
+**Mapbox SDK versions:** 
```

Comment:
Do you think it would be handy to know Xcode version? Maybe at least if they're testing with a beta release.

Reply:
Also, I wonder if we can provide examples of what we'd like to see here, or perhaps a link to a wiki page with a few examples.

Reply:
It would be helpful to write up a wiki page describing how to write a good, actionable issue for this repository (or for Mapbox’s iOS projects in general).